cmd/test/test-barcerola.yml

```diff
@@ -8,3 +8,45 @@ review:
     - <<: *web_base
       name: web
       memory: 256
+
+environments:
```

Comment:
not a big deal but file name has spelling error: `barcerola` 😅

Reply:
thanks fixed it